### PR TITLE
[zh-cn]: update the translation of `Document.exitFullscreen()` method

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/web/api/document/exitfullscreen/index.md [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
-document.exitFullscreen();
-```
+
+### 参数
+
+无。
+
+### 返回值
+
+一个 {{jsxref("Promise")}}，在 {{Glossary("user agent")}} 完成退出全屏模式后解决。如果在尝试退出全屏模式时发生错误，promise 的 `catch()` 处理程序将被调用。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/document/exitfullscreen/index.md:24]:
```suggestion
一个 {{jsxref("Promise")}}，在{{Glossary("user agent", "用户代理")}}完成退出全屏模式后被兑现。如果在尝试退出全屏模式时发生错误，Promise 的 `catch()` 处理程序将被调用。
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/document/exitfullscreen/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/document/exitfullscreen/index.md:24]: handler => 处理器

File: files/zh-cn/web/api/document/exitfullscreen/index.md [hunk 2]
```diff
@@ -36,12 +50,11 @@ document.onclick = function (event) {
...
-- [Using full-screen mode](/zh-CN/DOM/Using_full-screen_mode)
+- [Fullscreen API](/zh-CN/docs/Web/API/Fullscreen_API)
+- [全屏指南](/zh-CN/docs/Web/API/Fullscreen_API/Guide)
 - {{ domxref("Element.requestFullscreen()") }}
-- {{ domxref("Document.exitFullscreen()") }}
-- {{ domxref("Document.fullscreen") }}
 - {{ domxref("Document.fullscreenElement") }}
-- {{ cssxref(":fullscreen") }}
-- [`allowfullscreen`](/zh-CN/docs/Web/HTML/Element/iframe#allowfullscreen)
+- {{ cssxref(":fullscreen") }} and {{cssxref("::backdrop")}}
```
Comment (yin1999) [inline files/zh-cn/web/api/document/exitfullscreen/index.md:59]: and

Comment (tianyeeT) [inline files/zh-cn/web/api/document/exitfullscreen/index.md:59]:
```suggestion
- {{ cssxref(":fullscreen") }} 和 {{cssxref("::backdrop")}}
```

File: files/zh-cn/web/api/document/exitfullscreen/index.md (hunk 1 above)
Comment (tianyeeT) [inline files/zh-cn/web/api/document/exitfullscreen/index.md:24]:
```suggestion
一个 {{jsxref("Promise")}}，在{{Glossary("user agent", "用户代理")}}完成退出全屏模式后被兑现。如果在尝试退出全屏模式时发生错误，promise 的 `catch()` 处理器将被调用。
```

File: files/zh-cn/web/api/document/exitfullscreen/index.md [hunk 3]
```diff
@@ -36,12 +50,11 @@ document.onclick = function (event) {
 
 {{Compat}}
 
-## 参阅
+## 参见
 
-- [Using full-screen mode](/zh-CN/DOM/Using_full-screen_mode)
+- [Fullscreen API](/zh-CN/docs/Web/API/Fullscreen_API)
+- [全屏指南](/zh-CN/docs/Web/API/Fullscreen_API/Guide)
```
Comment (yin1999) [inline files/zh-cn/web/api/document/exitfullscreen/index.md:56]:
```suggestion
- [全屏 API](/zh-CN/docs/Web/API/Fullscreen_API)
- [全屏 API 指南](/zh-CN/docs/Web/API/Fullscreen_API/Guide)
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**